web/src/components/base/Button.tsx

```diff
@@ -16,6 +16,7 @@ type ButtonProps = {
   label: string
   type?: 'submit' | 'button'
   disabled?: boolean
+  dir?: string
```

Comment:
🔧 Why did you choose to make this configurable? This is never set to anything else but `auto`.

Reply:
Please remove this property if its not needed to set this to something else than `auto`.